### PR TITLE
social: add doc for JStackTemplate.create

### PR DESCRIPTION
**Pull request by rjeczalik**

### CODE REVIEW

File: workers/social/lib/social/models/computeproviders/stacktemplate.coffee [hunk 1]
```diff
@@ -162,6 +162,26 @@ module.exports = class JStackTemplate extends Module
...
+  #
+  # @example api
+  #
+  #   {
+  #     "template": "{\"provider\": {\"aws\": {}}}",
+  #     "title": "My AWS template",
+  #     "credentials": {}
+  #   }
+  #
+  @create = ->
```
Comment (rjeczalik) [inline workers/social/lib/social/models/computeproviders/stacktemplate.coffee]: /cc @gokemn

**Review by cihangir — APPROVED** [review]

File: workers/social/lib/social/models/computeproviders/stacktemplate.coffee (hunk 1 above)
Comment (gokmen) [inline workers/social/lib/social/models/computeproviders/stacktemplate.coffee]: actually, this should work;

```
...
"template": {
  "provider": {
    "aws": {}
  }
},
...
```

**Review by gokmen — APPROVED** [review]

File: workers/social/lib/social/models/computeproviders/stacktemplate.coffee (hunk 1 above)
Comment (rjeczalik) [inline workers/social/lib/social/models/computeproviders/stacktemplate.coffee]: I think it's not the same - the template field is expected to be a string, meaning it's stringified JSON. In your example template is going to be an object, isn't it?

Comment (rjeczalik) [inline workers/social/lib/social/models/computeproviders/stacktemplate.coffee]: The requirement for template to be a stringified JSON is that jStackTemplate.RawContent field is a string.

Comment (gokmen) [inline workers/social/lib/social/models/computeproviders/stacktemplate.coffee]: ah ok, you're right. I thought there was an issue with doc parser and you tried to fix that one. ignore my comment then.

**Pull request merged**